project.clj

```diff
@@ -12,7 +12,9 @@
   :exclusions [org.clojure/clojure]
 
   :dependencies [[org.clojure/clojure "1.6.0"]
+                 [org.clojure/java.jdbc "0.3.7"]
                  [cheshire "5.3.1"]
+                 [org.postgresql/postgresql "9.3-1100-jdbc41"]
```

Comment:
It's not directly used. What it does is provide a postgresql JDBC driver for clojure.java.jdbc. Without it, clojure.java.jdbc is unable to connect to a postgres database.